.github/workflows/create-test-matrices/action.yml

```diff
@@ -2,10 +2,23 @@ inputs:
     language-name:
         description: "Language name"
         required: true
-    dispatch-run-full-matrix:
+        type: choice
+        options:
+            - java
+            - node
+            - python
+            - go
+            - C#
+    run-full-matrix:
         description: "Run the full matrix"
-        required: false
-        default: "false"
+        required: true
+        type: boolean
+    containers:
```

Comment:
This parameter is not in use in jobs. It was used to run tests on containers such as alpine musl or amazon linux for all clients.
You need to restore more those files

Reply:
```
git diff upstream/CI/TrrigerByLabel..7c67b6b96 -- .github
```